internal/bits/count_amd64.s

```diff
@@ -31,7 +31,7 @@ TEXT ·countByte(SB), NOSPLIT, $0-40
     SHRQ $8, DX
     SHLQ $8, DX
     ADDQ AX, DX
-    VPBROADCASTB value+24(FP), Z0
+    VPBROADCASTB BX, Z0
```

Comment:
How did you catch this? Would be neat if we had an automated checker for instructions like this.

Reply:
Actually it was picked up by CI in some ways, the Github runners seem to not always execute on the same CPU types, some runs crashed because the code was executing instructions unknown to the CPU (because the features weren't checked properly).

I agree tho, I think AVO auto-generates comments that say which CPU features are required by a function, but something more advanced that could run in CI would be great (e.g. looking at all instructions and answering they are not executed if CPU features are not checked).